File: app/src/main/java/com/bmco/cratesiounofficial/OnDependencyDownloadListener.java
package com.bmco.cratesiounofficial;

import com.bmco.cratesiounofficial.models.Dependency;

import java.util.List;

/**
 * Created by bertuswisman on 26/05/2017.
 */

public interface OnDependencyDownloadListener {
    public void onDependenciesReady(List<Dependency> dependency);
}

File: app/src/main/java/com/bmco/cratesiounofficial/interfaces/OnDependencyDownloadListener.kt
package com.bmco.cratesiounofficial.interfaces

import com.bmco.cratesiounofficial.models.Dependency

/**
 * Created by bertuswisman on 26/05/2017.
 */

interface OnDependencyDownloadListener {
    fun onDependenciesReady(dependency: List<Dependency>)
}
